api/catalog/api/examples/audio_requests.py

```diff
@@ -23,29 +23,39 @@
 audio_search_list_curl = "\n".join(
     f"""
 # Example {index}: Search for audio {purpose}
-curl {auth} "{origin}/v1/audio/?q={syntax}"
+curl \\
```

Comment:
The original string seems easier to read here.

Reply:
Yeah, this one I only broke to be consistent with the lines below. On the positive side, these multiline cURLs look very good and readable in the ReDoc site.